pkg/network/opencontrail/config.go

```diff
@@ -45,8 +45,9 @@ func NewConfig() *Config {
 		ApiPort:          8082,
 		DefaultProject:   "default-domain:default-project",
 		PublicNetwork:    "default-domain:default-project:Public",
-		PrivateSubnet:    "10.0.0.0/16",
-		ServiceSubnet:    "10.254.0.0/16",
+		PrivateSubnet:    "10.10.0.0/16",
+		ServiceSubnet:    "10.247.0.0/16",
```

Comment:
Please don't  commit this change.
You can override this in the command line by specifying: kube-network-manager -- --option=x